docs/docsite/rst/guide_vmware.rst

```diff
@@ -0,0 +1,62 @@
+Getting Started with VMware
+===========================
+
```

Comment:
Please add table of contents with:
`.. contents:: Topics`

Reply:
OK.

Reply:
No, we don't need a TOC here.